Resources/config/services.xml

```diff
@@ -6,10 +6,12 @@
 
     <parameters>
         <parameter key="lexik_jwt_authentication.jwt_encoder.class">Lexik\Bundle\JWTAuthenticationBundle\Encoder\JWTEncoder</parameter>
+        <parameter key="lexik_jwt_authentication.encoder.default.class">Lexik\Bundle\JWTAuthenticationBundle\Encoder\DefaultEncoder</parameter>
         <parameter key="lexik_jwt_authentication.jwt_manager.class">Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager</parameter>
         <parameter key="lexik_jwt_authentication.handler.authentication_success.class">Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationSuccessHandler</parameter>
         <parameter key="lexik_jwt_authentication.handler.authentication_failure.class">Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationFailureHandler</parameter>
-        <parameter key="lexik_jwt_authentication.openssl_key_loader.class">Lexik\Bundle\JWTAuthenticationBundle\Services\OpenSSLKeyLoader</parameter>
+        <parameter key="lexik_jwt_authentication.key_loader.openssl.class">Lexik\Bundle\JWTAuthenticationBundle\Services\KeyLoader\OpenSSLKeyLoader</parameter>
+        <parameter key="lexik_jwt_authentication.key_loader.seclib.class">Lexik\Bundle\JWTAuthenticationBundle\Services\KeyLoader\SecLibKeyLoader</parameter>
```

Comment:
I think we can avoid adding *new* parameters at least (but as we discussed, remove the old ones in a different PR)